content/en/docs/concepts/workloads/pods/pod-overview.md

```diff
@@ -74,9 +74,9 @@ Here are some examples of workload resources that manage one or more Pods:
 
 ## Pod templates
 
-Generally: controllers for workload resources use a Pod template (that you provide) to create the Pods for that workload. The controller is responsible for managing those Pods on your behalf.
+Generally: controllers for {{< glossary_tooltip text="workload" term_id="workload" >}} resources use a Pod template (that you provide) to create the Pods for that workload. The controller is responsible for managing those Pods on your behalf.
```

Comment:
@sftim , I went through the page again, carefully. I think the text could be pared down so that the message is more succinct.

Line 32, nit: replace e.g. with For example, (e.g., run multiple instances)
Line 32, nit: duplicate reference and text to internal link, Pods and Controllers. See line 62:

32:
Replicated Pods are usually created and managed as a group by a workload _controller_. See [Pods and controllers](#pods-and-controllers) for more information.

62:
Thus, while it is possible to use Pod directly, it's far more common in Kubernetes to manage your pods using a controller. See [Pods and controllers](#pods-and-controllers) for more information on how Kubernetes uses controllers to implement workload scaling and healing.

Line 54: This paragraph uses capitalized `Node` when I think `node` would do.
Line 59: Would you clarify/rewrite the note, but is an environment (Pod) the containers run in `and persists until _it_ is deleted`.
Suggested edit: Restarting a container in a Pod should not be confused with restarting a Pod. A Pod is not a process, but an environment for a running container. A Pod persists until it is deleted.